app/models/delete_queue_job.rb

```diff
@@ -71,6 +71,7 @@ def perform
         # once a user adds another metadata file
         ClusterGroup.where(study_id: study.id).each do |cluster_group|
           delete_subsampled_data(cluster_group)
+          cluster_group.update(subsampled: false)
```

Comment:
these two lines seem like they'd be better placed in a delete_subsampled_data function on the cluster_group model

Reply:
Fair point.